parametric_stellarator.py

```diff
@@ -470,9 +528,24 @@ def parametric_stellarator(
     if include_graveyard == True:
         components = graveyard(vmec, offset, components)
 
+    # Conditionally build magnet coils
+    if include_magnets == True:
```

Comment:
```suggestion
    if include_magnets:
```

Reply:
Yes, it is getting a bit cumbersome. One idea is to have different aspects of the model (e.g., export, magnets, source mesh later on, etc.) passed as their own dictionaries